modules/wizard/view-wizard-step-media.php

```diff
@@ -7,12 +7,10 @@
  * @since 2.7
  */
 
-if ( ! defined( 'ABSPATH' ) ) {
-	exit; // Don't access directly.
-}
+defined( 'ABSPATH' ) || exit; // Don't access directly.
 
 $default_options = PLL_Install::get_default_options();
-$options = wp_parse_args( get_option( 'polylang' ), $default_options );
+$options = wp_parse_args( (array) get_option( 'polylang' ), $default_options );
```

Comment:
Shouldn't we pass Polylang options to this view (like the other Wizard views)  instead of using `get_option()`?

Reply:
Done in 7eb6ea6e27ab6daf91e444750528577a1b7afa4e